src/js/server/kettleValidation.js

```diff
@@ -5,6 +5,17 @@ var gpii  = fluid.registerNamespace("gpii");
 
 fluid.registerNamespace("gpii.schema.kettle.validator");
 
+/**
+ *
+ * Validate a request payload according to a GSS Schema.  Fulfills the contract for a `kettle.middleware` `handle`
+ * invoker.
+ *
+ * @param {Object} kettleValidator - A `gpii.schema.kettle.validator` instance that has a schema and rules about which part of the payload should be validated.
```

Comment:
I'm tending to prefer putting at the very least {Component} in these cases but also more helpfully, e.g. {gpii.schema.kettle.validator}

Reply:
I went through all JSDocs usages of `{Object}` and updated all of them.  In a few cases I did have the grades in the description, I moved most of those to the type.  I left a single redundant one on this line rather than contort the description to avoid mentioning the grade again.